tests/data-files/data-files-add.spec.js

```diff
@@ -13,69 +13,18 @@ test.describe('test Add button', async () => {
       await page.getByRole('link', { name: 'Data Files' }).click();
     })
 
-    test('test create new folder', async ({ page, portal, environment, baseURL }) => {
-      // Navigate to Create Folder dialog
-      await page.getByRole('button', { name: '+ Add'}).click();
-      await page.locator('button:has-text(" Folder")').click();
-
-      // Test folder creation
-      const folderName = page.locator('[name="dirname"]');
-      await folderName.fill('testFolder1');
-      await page.locator('button:has-text("Create Folder")').click();
+    test('test create new folder', async ({ fileOperations }) => {
+      await fileOperations.createFolder('E2EtestFolder1');
     });
 
-    test('test upload file', async ({ page, portal, environment, baseURL }) => {
-      // Navigate to Create File dialog
-      await page.getByRole('button', { name: '+ Add'}).click();
-      await page.locator('button:has-text("Upload")').click();
-
-      // Test upload file
-      const [fileChooser] = await Promise.all([
-        page.waitForEvent('filechooser'),
-        page.getByText('Select File(s)').click(),
-      ]);
-      var buf = Buffer.from('this is a test', 'utf8');
-      await fileChooser.setFiles(
-        [
-            {"name": "testFile1", "mimeType": "text/plain", "buffer": buf},
-        ],
-      );
-
-      await page.locator('button:has-text("Upload Selected")').click();
-      const statusMessage = page.locator('dialog span.badge');
-      expect (await statusMessage.isVisible());
-
-      await page.waitForTimeout(5000);
-      await page.getByRole('dialog').getByRole('button', { name: 'Close' }).click();
+    test('test upload file', async ({ page, fileOperations }) => {
+      await fileOperations.uploadFile('E2EtestFile1', 'File content.');
+      expect (await page.getByRole('checkbox', { name: 'select file E2EtestFile1' }).isVisible());
```

Comment:
The test is failing for me here, I think because the modal isn't being closed after the upload happens.
```suggestion
      await page.getByRole('dialog').getByLabel('Close').click();
      expect (await page.getByRole('checkbox', { name: 'select file E2EtestFile1' }).isVisible());
```

Reply:
After running with the change, this expect is still failing because because it matches 9 selectors.

Reply:
Reviewing this more, it seems like it was an issue with the Data FIles listing of the user. I'm marking this as resolved